.github/workflows/build-and-publish.yml

```diff
@@ -2,6 +2,8 @@ on:
   push:
     branches:
       - main
+    tags:
```

Comment:
this is not necessary anymore, you should get a build in S3 in the comments

Reply:
amazing